html/admin/index.html

```diff
@@ -129,10 +129,24 @@ <h1>Wireless Access Point</h1>
       </div>
       <p class="lead">Configure the channel used by the Wireless Access Point</p>
           <form class="simple_form form-inline" id="form_channel">
-              <div class="form-group required"><label class="required sr-only" for="channel"><abbr title="required">*</abbr> Channel</label><input class="string required form-control" id="input_channel" name="channel" placeholder="Enter Channel" type="text"></div>
-
-              <input class="btn btn-default" name="update_channel" type="submit" value="Update">
-              <div style="display:none" id="update_channel_success">✓</div>
+            <div class="form-group required"><label class="required sr-only" for="channel"><abbr title="required">*</abbr> Channel</label>
+              <select class="string required form-control" id="input_channel" name="channel">
+		<!-- Ultimately populate this with the acceptable channels, which would be exposed via the API -->
+		<option value="1">1</option>
```

Comment:
Since the value and label of the select options are the same, you can drop the 'value' attribute so you have <option>1</option>